docs/cloud-common.md

```diff
@@ -1,6 +1,30 @@
-# Set up Ark with your cloud provider
+# Set up Ark on your platform
 
-To run Ark with your cloud provider, you specify provider-specific settings for the Ark server. In version 0.7.0 and later, you can run Ark in any namespace, which requires additional customization. See [Run in custom namespace][3].
+You can run Ark with a cloud provider or on-premises. For detailed information about the platforms that Ark supports, see [Compatible Storage Providers][99].
+
+In version 0.7.0 and later, you can run Ark in any namespace, which requires additional customization. See [Run in custom namespace][3].
+
+In version 0.9.0 and later, you can use Ark's integration with restic, which requires additional setup. See [restic instructions][20].
+
+## Set up Ark on-premises
```

Comment:
Note for myself: probably move this to the bottom, I think it makes sense to keep cloud-provider instructions first